### DISCUSSION TITLE
Facebook Messenger Integration (on premise installation)

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4840] [Self hosted]
Facebook ask me a description and a video of the app, someone can help me?
Someone did it succesfully? There is a simple tutorial?
![image](https://user-images.githubusercontent.com/62311149/173196491-8fbecd64-04a6-4b65-9a4c-ade6e9df2f61.png)

**Comment by sojan-official** [comment]
please refer : https://github.com/orgs/chatwoot/discussions/3397
